apply_license.py

```diff
@@ -1,6 +1,13 @@
+#!/usr/bin/env python
 import os
+import time
+import re
+import subprocess
 
-LICENSE = '''Copyright 2020 The Defold Foundation
+RE_LICENSE = r"(.*?\n?\r?)[/#;-]?[/#;-]\sCopyright (\d\d\d\d) The Defold Foundation.*specific language governing permissions and limitations under the License.(.*)"
```

Comment:
Expression matching an existing license in a file (used when updating a license)